javascript/builtins/Atomics.json

```diff
@@ -64,7 +91,7 @@
               "version_added": false
             },
             "safari": {
-              "version_added": "10.1"
+              "version_added": false
```

Comment:
Should this be ``false`` or a ``version_removed``?

Reply:
I don't actually know the version this will be removed, so I thought it's best to just mark it false for now. It could be `version_removed: true` though. hmm...

Reply:
Updated